docs/reference/project_package.md

```diff
@@ -0,0 +1,11 @@
+## Packaging data
+
+OC4IDS data must be published in a project package which acts an envelope for the data and contains important metadata.
```

Comment:
contains -> adds

Reply:
Fixed in https://github.com/open-contracting/infrastructure/pull/89/commits/381b67bbdee4987956315a5b528d5b5f8538634e